api/src/main/resources/db/migration/V3__MoveMemeTable.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE memes;
+
+CREATE TABLE AbstractMemery.memes(
+    ID SERIAL NOT NULL PRIMARY KEY,
+    memeName VARCHAR(200) UNIQUE,
```

Comment:
Do we really care if the name is unique?  The DB will create a unique PK for it.

Reply:
That's a good point. I pushed up another migration to fix that